<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>orderwindow</class>
 <widget class="QDialog" name="orderwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser_4">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>70</y>
     <width>811</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#f6f5ef;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-bottom:1px solid #ddd;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="coffeeButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>15</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
font-weight:400;
}
QPushButton:hover{

}</string>
   </property>
   <property name="text">
    <string>COFFEE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="teaButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>15</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;
font-weight:400;
}
QPushButton:hover{

}</string>
   </property>
   <property name="text">
    <string>TEA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/logo1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>801</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#2c2a29;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>110</y>
     <width>841</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1b3c35;
border:none;</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/coffee1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgba(255,255,255,0);
border:1px solid #fff;
opacity:0.5;
color:#fff;
}
QPushButton:hover{
background-color:rgba(255,255,255,0.5);
}</string>
   </property>
   <property name="text">
    <string>◀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgba(255,255,255,0);
border:1px solid #fff;
opacity:0.5;
color:#fff;
}
QPushButton:hover{
background-color:rgba(255,255,255,0.5);
}</string>
   </property>
   <property name="text">
    <string>▶</string>
   </property>
  </widget>
  <widget class="QLabel" name="contentLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff; font-weight:300; font-size:14px;</string>
   </property>
   <property name="text">
    <string>진한 에스프레소에 시원한 정수물과
얼음을 더하여 스타벅스의 깔끔하고 강렬한
에스프레소를 가장 부드럽고 시원하게 즐길 수 있는 커피</string>
   </property>
  </widget>
  <widget class="QLabel" name="menuLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff; font-weight:600</string>
   </property>
   <property name="text">
    <string>아이스 카페 아메리카노</string>
   </property>
  </widget>
  <widget class="QPushButton" name="listButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border:1px solid #1b3c35;
opacity:0.5;
color:#1b3c35;
}
QPushButton:hover
{
background-color:#1b3c35;
color:#fff;
}</string>
   </property>
   <property name="text">
    <string>주문 담기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="orderButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border:1px solid #1b3c35;
opacity:0.5;
color:#1b3c35;
}
QPushButton:hover
{
background-color:#1b3c35;
color:#fff;
}</string>
   </property>
   <property name="text">
    <string>주문 하기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#006636;
font-weight:700;font-size:18px;</string>
   </property>
   <property name="text">
    <string>Espresso</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
